persist/adapter.go

```diff
@@ -48,6 +48,8 @@ type Adapter interface {
 	LoadPolicy(model *model.Model) error
 	// SavePolicy saves all policy rules to the storage.
 	SavePolicy(model *model.Model) error
+	// ClearPolicy remove all policy rules from the storage.
+	ClearPolicy() error
```

Comment:
This PR adds `ClearPolicy()` to the adapter interface and will cause compile error, which is a BREAKING change for ALL current adapter implementations.

Reply:
Yes, I know, but https://github.com/casbin/casbin/pull/587 has caused the adapter to be re-implemented, because the model has become private, we can only operate through public func, I think it’s time finished this feature in casbin v3

Reply:
I submitted a PR #590 for compatible with V2. @dovics

Reply:
It is not fully compatible with the v2 adapter, because the way of operating the model needs to be changed from `model["p"]["p"]` to using public functions.

And do you think this modification is unnecessary? It may cause model is inconsistent with adapter.

Reply:
@dovics  please refer to https://github.com/casbin/casbin/pull/592#discussion_r502783238.